app/src/main/java/org/luxoft/sdl_core/JavaToNativeBleAdapter.java

```diff
@@ -13,15 +13,18 @@
     public static final String TAG = JavaToNativeBleAdapter.class.getSimpleName();
     private static final int WRITE_ID = 1;
     private static final int READ_ID = 2;
+    private static final int WRITE_CONTROL_ID = 3;
 
     Handler mHandler;
     BleWriter mWriter;
+    BleWriter mControlWriter;
     BleReader mReader;
     BleAdapterMessageCallback mCallback;
     private final Context mContext;
 
     JavaToNativeBleAdapter(Context context){
-        mWriter = new BleLocalSocketWriter();
+        mWriter = new BleLocalSocketWriter("./localBleWriter");
```

Comment:
Just suggestion: it might be better to store such addresses (like ./localBleWriter) as constants in some good place, which may be accessible for native and java sides. Or create separate constants for each side for now, because these hard-coded values don't look nice.

Reply:
As for now, we can make it private static String in this class

Reply:
done in 66d56ca414ac2d17dea823a2186a8d3f2569ec83